src/constants/joyride/index.js

```diff
@@ -0,0 +1,5 @@
+export {initialStepSettings} from './joyrideSettings/tourSettings';
+export {steps as profileSteps} from './joyrideTooltips/profile';
+export {steps as startPageSettings} from './joyrideTooltips/startPage';
+export {steps as taskPreviewSettings} from './joyrideTooltips/taskPreview';
+export {steps as taskTrainSettings} from './joyrideTooltips/taskTrain';
```

Comment:
As far as I know short form of reexport brakes navigation in Webstorm, maybe we can use long variant? 
```
import {taskTrainSettings} from './joyrideTooltips/taskTrain';
import {taskPreviewSettings} from './joyrideTooltips/taskPreview';

export default {
    taskTrainSettings,
    taskPreviewSettings
}